packages/apps-config/package.json

```diff
@@ -29,7 +29,7 @@
     "@zeitgeistpm/type-defs": "0.2.7",
     "@zeroio/type-definitions": "0.0.6",
     "lodash": "^4.17.21",
-    "moonbeam-types-bundle": "1.2.5",
+    "moonbeam-types-bundle": "latest",
```

Comment:
Would prefer a specific version here. This package is used by others as well.

Reply:
yes ofc sorry about that